lxd/network/openvswitch/ovn.go

```diff
@@ -2047,3 +2082,154 @@ func (o *OVN) LogicalRouterPeeringDelete(opts OVNRouterPeering) error {
 
 	return nil
 }
+
+// GetHardwareAddress gets the mac address of the ovn managed network.
+func (o *OVN) GetHardwareAddress(chassisGroup OVNChassisGroup) (string, error) {
+	nameFilter := fmt.Sprintf("name=%s-lr-lrp-ext", chassisGroup)
+	data, err := o.nbctl("--data=bare", "--format=csv", "find", "Logical_Router_Port", nameFilter)
```

Comment:
As you're only extracting a single value, there's no need to use a csv parser here, you can just ask ovn-nbctl for the specific field using `--column=` arg and use `--data=bare` with `--no-headings`. See https://github.com/lxc/lxd/blob/master/lxd/network/openvswitch/ovn.go#L1312 for an example.